source/docs/appendix/gallery.rst

```diff
@@ -8,6 +8,15 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
 2022-2023 Power Play
 ------------------------
 
+11329 I.C.E. Robotics 
+^^^^^^^^^^^^^^^^^^^^^
+
+- `CAD <https://grabcad.com/library/2023-ftc-power-play-cad-team-11329-i-c-e-robotics-glacier-1>`__
+- `Robot Reveal Video <https://youtu.be/deOm05iy3Ak>`__
+- `Behind the Bot <https://youtu.be/Bhwif_vSumw>`__
```

Comment:
Please replace these with the full youtube.com links instead of the short links.